cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ConfigurationInstanceHandlers.java

```diff
@@ -206,7 +206,7 @@ public static void post(HttpServletRequest request, HttpServletResponse response
      * @throws IOException
      */
     public static void put(HttpServletRequest request, HttpServletResponse response, boolean v1) throws IOException {
-        response.setContentType("json;charset=utf-8");
+        response.setContentType("application/json; charset=utf-8");
```

Comment:
Is this a fix? Should be mentioned in CNR file?

Reply:
NTC

Reply:
Nobody is using that API